### PR TITLE
[Exe, Ver] Increase event size limit for system chunk

### PR DESCRIPTION
**Pull request by ramtinms**
This PR: 
- moves system chunk config into a single location 
- increases the limit for events to a higher number

### CODE REVIEW

File: engine/execution/computation/computer/computer.go [hunk 1]
```diff
@@ -26,6 +26,8 @@ import (
 	"github.com/onflow/flow-go/utils/logging"
 )
 
+const SystemChunkEventCollectionMaxSize = 1000000
```
Comment (jordanschalm) [inline engine/execution/computation/computer/computer.go:29]:
```suggestion
const SystemChunkEventCollectionMaxSize = 1_000_000
```

File: engine/execution/computation/computer/computer.go [hunk 2]
```diff
@@ -52,6 +54,17 @@ type blockComputer struct {
 	committer      ViewCommitter
 }
 
+func SystemChunkContext(vmCtx fvm.Context, logger zerolog.Logger) fvm.Context {
```
Comment (jordanschalm) [inline engine/execution/computation/computer/computer.go:57]: 👍

**Review by jordanschalm — APPROVED** [review]
🎸

**Review by AlexHentschel — COMMENTED** [review]
🧹 ✨

**Review by Kay-Zee — APPROVED** [review]

**Comment by Kay-Zee** [comment]
bors merge

**Comment by Kay-Zee** [comment]
bors merge

**Pull request merged**